tests/formats/test_bctex.py

```diff
@@ -4,6 +4,11 @@
 from mercury_engine_data_structures import dread_data, samus_returns_data
 from mercury_engine_data_structures.formats.bctex import Bctex
 
+dread_exclusions = [
+    # exceptionally slow file
+    "textures/system/fx/tests/watertank/textures/watertank_col.bctex",
+]
+
```

Comment:
I might bundle this into dread_210_ignore just so we don't have as many arrays in here. I also realize dread_210_ignore is a bit misleading since those are always ignored. something like

```
dread_exclusions = [
    # exceptionally slow file
    "textures/syustem/fx/tests/watertank/textures/watertank_col.bctex",

    # these bytematch existing textures (originally prefixed by "textures/"). likely put in the bossrush pkgs for performance. 
    <exclusions>
]
```